resources/leiningen/new/kraken_works/PROJECT@.service.template

```diff
@@ -26,11 +24,11 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 
 ExecStart=/bin/bash -c 'docker run --name ${CONTAINER} \
   --link rabbitmq:rabbitmq \
```

Comment:
I think we should drop the `--link rabbitmq:rabbitmq` too (Docker links are deprecated) and replace it with `--dns $COREOS_PRIVATE_IPV4`, `--env RABBITMQ_PORT_5672_TCP_ADDR=rabbitmq.service.consul`, and `--env RABBITMQ_PORT_5672_TCP_PORT=5672`. That's what a typical service template for a live component does today anyway.